python/artm/model.py

```diff
@@ -662,6 +667,8 @@ def fit_transform(self, topic_names=None, remove_theta=False):
           default=None (means all topics)
           remove_theta (bool): flag indicates save or remove Theta from model
           after extraction, default=False
+          use_doc_titles (bool): name columns of theta dataframe by document titles,
```

Comment:
Thank you for fixing #308 , this issue had been opened for way too long :)
Have you consider other ways of customizing the behavior? The issue with adding a Boolean flag to `fit_transform` method is that 
* that this flag needs to be in both methods (`get_theta` and `transform`). 
* most likely users want to configure the behavior once and for all calls. 
I would consider adding a string property on `ARTM` class, named for example `ThetaMatrixColumnsType`. Expected values should be `id` and `title`.